app/src/ui/stash-changes/overwrite-stashed-changes-dialog.tsx

```diff
@@ -0,0 +1,57 @@
+import React = require('react')
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+import { Repository } from '../../models/repository'
+import { Branch } from '../../models/branch'
+import { Dispatcher } from '../dispatcher'
+import { ButtonGroup } from '../lib/button-group'
+import { Button } from '../lib/button'
+import { UncommittedChangesStrategy } from '../../models/uncommitted-changes-strategy'
+
+interface IOverwriteStashedChanges {
+  readonly dispatcher: Dispatcher
+  readonly repository: Repository
+  readonly branchToCheckout: Branch
+  readonly onDismissed: () => void
+}
+
+export class OverwriteStashedChanges extends React.Component<
+  IOverwriteStashedChanges,
+  {}
+> {
+  public render() {
+    const title = __DARWIN__
+      ? 'Are You Sure You Want To Overwrite Your Current Stash?'
+      : 'Are you sure you want to overwrite your current stash?'
```

Comment:
This is a little meaty for a title. Could we incorporate this message into the dialog contents and make the title be something like `Overwrite Stash?`

Reply:
I decided to get rid of this content entirely since putting it in the dialog content area would be redundant.